docs/topics/db/models.txt

```diff
@@ -954,16 +954,43 @@ extend the parent's :ref:`Meta <meta-options>` class, it can subclass it. For ex
 
 Django does make one adjustment to the :ref:`Meta <meta-options>` class of an abstract base
 class: before installing the :ref:`Meta <meta-options>` attribute, it sets ``abstract=False``.
-This means that children of abstract base classes don't automatically become
-abstract classes themselves. Of course, you can make an abstract base class
-that inherits from another abstract base class. You just need to remember to
-explicitly set ``abstract=True`` each time.
+This means that children of abstract base classes don't automatically become abstract classes
+themselves. You can make an abstract base class that inherits from another abstract base
+class; you just need to remember to explicitly set ``abstract=True`` each time.
 
 Some attributes won't make sense to include in the :ref:`Meta <meta-options>` class of an
 abstract base class. For example, including ``db_table`` would mean that all
 the child classes (the ones that don't specify their own :ref:`Meta <meta-options>`) would use
 the same database table, which is almost certainly not what you want.
 
+Due to the way Python inheritance works, if a child class inherits from multiple abstract base
+classes, only the :ref:`Meta <meta-options>` options from the first listed class will be
+inherited by default. To inherit :ref:`Meta <meta-options>` options from multiple abstract
+base classes, or from a base class not listed first, you must explicitly declare the
+:ref:`Meta <meta-options>` inheritance::
+
+    from django.db import models
+
+    class CommonInfo(models.Model):
+        name = models.CharField(max_length=100)
+        age = models.PositiveIntegerField()
```

Comment:
As these fields are already defined in the above examples, we could just do `# ...`

Reply:
I think it's best to keep them here - explicit, and people don't read docs pages top to bottom but often only the ref'd section.

Reply:
Agreed, also I forgot to add, since these fields are not mention in this section of the doc, it's not important to have them in the examples.